fuzz/dex/v0.1/fuzzDexExecutorEventEnterFarm.go

```diff
@@ -0,0 +1,72 @@
+package dex
+
+import (
+	"errors"
+	"fmt"
+
+	vmi "github.com/ElrondNetwork/elrond-go/core/vmcommon"
+)
+
+func (pfe *fuzzDexExecutor) enterFarm(user string, farm Farm, amount int, statistics *eventsStatistics) error {
+
+	output, _ := pfe.executeTxStep(fmt.Sprintf(`
+	{
+		"step": "scCall",
+		"txId": "stake",
+		"tx": {
+			"from": "%s",
+			"to": "%s",
+			"value": "0",
+			"function": "enterFarm",
+			"esdt": {
+				"tokenIdentifier": "str:%s",
+				"value": "%d"
+			},
+			"arguments": [],
+			"gasLimit": "100,000,000",
+			"gasPrice": "0"
+		}
+	}`,
+		user,
+		farm.address,
+		farm.farmingToken,
+		amount,
+	))
+	if output == nil {
```

Comment:
check the error, not the output, if there is no error, the output is never nil

Reply:
done